src/instructlab/configuration.py

```diff
@@ -201,7 +201,7 @@ class _serve_vllm(BaseModel):
     gpus: Optional[int] = Field(default=None, description="Number of GPUs to use.")
     # arguments to pass into vLLM process
     vllm_args: list[str] | None = Field(
-        default_factory=list,
+        default_factory=list,  # type: ignore
```

Comment:
yes but please add comments with links to issues / patches that will allow us to remove these ignores

Reply:
yes, added now!
Thank you for the investigative work!